lib/matplotlib/axes/_axes.py

```diff
@@ -5801,9 +5801,11 @@ def hist(self, x, bins=None, range=None, normed=False, weights=None,
         normed : boolean, optional
             If `True`, the first element of the return tuple will
             be the counts normalized to form a probability density, i.e.,
-            ``n/(len(x)`dbin)``, i.e., the integral of the histogram will sum
-            to 1. If *stacked* is also *True*, the sum of the histograms is
-            normalized to 1.
+            the integral of the histogram will sum to 1. So, the normalized
```

Comment:
I hope it is right now from your point of view. I flipped and shortened the explanation. I think it is in the limit to be useful to avoid new confusion about the normalization method that is applied. Thanks.